Documentation/doc/resources/1.9.3/footer.html

```diff
@@ -0,0 +1,21 @@
+<!-- HTML footer for doxygen 1.8.20-->
+<!-- start footer part -->
```

Comment:
@sloriot 

I think some updating would be in place ...
Probably also valid for other files.

Reply:
I only did a copy/paste for now. I can do the update once doxygen is officially out (to avoid redoing it again).